taskcluster/scripts/toolchain/android-gradle-dependencies.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+set -ex
+
+function get_abs_path {
+    local file_path="$1"
+    echo "$( cd "$(dirname "$file_path")" >/dev/null 2>&1 ; pwd -P )"
+}
+
+CURRENT_DIR="$(get_abs_path $0)"
+PROJECT_DIR="$(get_abs_path $CURRENT_DIR/../../../..)"
+
+pushd $PROJECT_DIR
+
+. taskcluster/scripts/toolchain/android-gradle-dependencies/before.sh
+
+GRADLE_FLAGS='--no-daemon -PgoogleRepo=http://localhost:8081/nexus/content/repositories/google/ -PjcenterRepo=http://localhost:8081/nexus/content/repositories/jcenter/'
+
+./gradlew $GRADLE_FLAGS ktlint detekt
```

Comment:
The rest of the script is different from r-b. We basically read `.buildconfig.yml` to get all components and build each one of them. It currently takes 2 hours. I wonder if using `--parallel` is a good thing here. I know it caused some issues in the past. What do you think @pocmo?

Reply:
I wonder why call gradle for every component instead of once to build everything like previously?

Yeah, `--parallel` has been causing intermittent issues in tests. I think that would be particularly annoying when this long running task fails.

Reply:
I ended using `--parallel` which shows the best results in terms of performance and reliability (I tried other ways and it didn't work out). I made the tests not fail the cache anymore, so I think it won't be too annoying.